### ISSUE TITLE
Object classes of Pre-trained Mask R-CNN

### ISSUE THREAD

**Issue by xxxmian** [alfworld/alfworld#12]
Hi, thanks for your amazing work. 
I have some questions about the pre-trained detector. In the supplementary material, the object categories  is 105.

![image](https://user-images.githubusercontent.com/34122441/105679120-f7767c00-5f28-11eb-91bb-773035d0a29f.png)

However, the pretrained mask r-cnn detector detect only 73 classes. I have checked the [constants.py](https://github.com/alfworld/alfworld/blob/aeea37e01f0cec0447b12ef5ae39d0ab13fbc003/alfworld/gen/constants.py#L1201) and guess  the  73  classes belongs to   OBJECTS_DETECTOR, the leaved 32 classes is STATIC_RECEPTACLES.

I am wondering would you provide the pre-trained detector with ALL 105 object classes ?
Thanks.

**Comment by MohitShridhar** [comment]
@xxxmian thanks for pointing this out! That's right, it's 73 classes. We will update this in the camera-ready version.

Here are the pre-trained models for [objects, receptacles, and all object categories](https://drive.google.com/drive/folders/1eHzFw-a7IIX5iV-tQTu6fXKIuxxha6ag?usp=sharing). Also added a new section in the [README](https://github.com/alfworld/alfworld/blob/master/alfworld/data/README.md#pre-trained-models).

Note that the all objects model performs poorly, probably due to imbalanced classes.
